public-dashboard-client/src/viz-recidivism-rates/RecidivismRatesChart.js

```diff
@@ -128,12 +128,16 @@ export default function RecidivismRatesChart({ data }) {
                     strokeWidth: 2,
                   };
                 }}
-                lines={chartData}
+                lines={data}
                 points={points}
                 pointStyle={{
                   fill: pointColor,
                   r: 5,
                 }}
+                renderKey={(d) =>
+                  // if it has a label, it's a line; if not, it's a point
+                  d.label || `${d.releaseCohort}-${d.followupYears}`
+                }
```

Comment:
It's not super clear to me whats going on here. This chart only has lines, correct? Or are points in a future feature?

Reply:
See my comment about this in the follow-up PR. It might be nice to add a label to the points instead.

Reply:
no there are also points, you can see them when you hover over an item in the legend or in the cohort selection dropdown

Reply:
yeah good call ... why don't we just deal with this issue in that PR then? If I change it here I'll just wind up with a merge conflict anyway